src/app/api/attendees/create/route.ts

```diff
@@ -0,0 +1,22 @@
+import { NextResponse } from 'next/server'
+import { cookies } from 'next/headers'
+import { createServerClient } from '@/utils/supabase'
+
+export async function POST(request: Request) {
+  const cookieStore = cookies()
+  const supabase = createServerClient(cookieStore)
+  const body = await request.json()
+
+  const { data, error } = await supabase
+    .from('attendees')
+    .insert([body])
+    .select()
```

Comment:
I'm not sure why the select is here. Is it necessary?

Reply:
Removed select.